api/_topic_map.yml

```diff
@@ -1336,19 +1336,24 @@
 
   /arm64/twistcli:
     get:
-      summary: Download ARM64 twistcli
+      summary: Download ARM64 twistcli for Linux OS
       description: !include ../descriptions/util/twistcli_arm64_get.md
 
   /twistcli:
     get:
-      summary: Download twistcli for GNU/Linux
+      summary: Download twistcli for Linux OS
       description: !include ../descriptions/util/twistcli_get.md
 
   /osx/twistcli:
     get:
       summary: Download twistcli for OS X
       description: !include ../descriptions/util/osx_twistcli_get.md
 
+  /osx/arm64/twistcli:
```

Comment:
@ssugandh is this for Mac? if so, maybe we should mention "MacOS" instead of "OS X"?

Reply:
Sure @grevach, I updated.